### PR TITLE
Chore/contributing guidelines

### PR DESCRIPTION
**Pull request by nellyk**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,145 +1,118 @@
...
-The above syntax helps reduce verbosity in the _subject_. In comparison, consider the following example:
+Runs `eslint` **and** unit tests on the current branch.
 
-```bash
-fix(transaction): missing quantity field in txn details
-```
+`npm posttest`
 
-Another scenario for scope is using a `route/scope` (or `context/scope`) syntax. This would be useful when a commit only affects a particular instance of code that is used in multiple places.
+Runs linting on the current branch and checks that the commits follow [conventional commits](https://www.conventionalcommits.org/)
```
Comment (JAdshead) [inline CONTRIBUTING.md]: this just lints the lockfile
https://github.com/americanexpress/one-app-dev-proxy/blob/master/package.json#L13

conventional commits are run as part of git hooks

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,145 +1,118 @@
...
-```
+`npm test`
 
-The above syntax helps reduce verbosity in the _subject_. In comparison, consider the following example:
+Runs `eslint` **and** unit tests on the current branch.
 
-```bash
-fix(transaction): missing quantity field in txn details
-```
+`npm posttest`
```
Comment (mtomcal) [inline CONTRIBUTING.md]: No `npm posttest` job.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,145 +1,118 @@
...
 
-#### Type
+   > replace `your-github-username` with your github username
 
-The *type* should always be lowercase as shown below.
+3. Install the dependencies by running
 
-##### Allowed `<type>` values:
+    ```bash
+    $ cd npm install
```
Comment (10xLaCroixDrinker) [inline CONTRIBUTING.md]:
```suggestion
    $ npm install
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,145 +1,118 @@
...
-Another scenario for scope is using a `route/scope` (or `context/scope`) syntax. This would be useful when a commit only affects a particular instance of code that is used in multiple places.
+Runs linting on the current branch and checks that the commits follow [conventional commits](https://www.conventionalcommits.org/)
 
-*Example*: Transactions may be shown in multiple routes/contexts, but a bug affecting transaction actions may only exist under the "home" route, possibly related to other code. In such cases, you could use the following format:
+## Submitting a new feature
 
-```bash
-fix(home/transactions): txn actions not working
-```
+When submitting a new feature request or enhancement of an existing features please review the following:-
```
Comment (10xLaCroixDrinker) [inline CONTRIBUTING.md]:
```suggestion
When submitting a new feature request or enhancement of an existing features please review the following:
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,145 +1,117 @@
...
 
-For bug fixes that address open issues, the footer should be formatted like so:
+Provide the system information which is not limited to the below:
 
-```bash
-Closes #17, #26
-```
-and for Pull Requests, use the format:
+- Browser (if applies) [e.g. chrome, safari]
+- Version of one-app-dev-proxy: [e.g. 5.0.0]
```
Comment (mtomcal) [inline CONTRIBUTING.md]: Add OS

Comment (nellyk) [reply CONTRIBUTING.md]: updated a110342

**Pull request merged**